components/layout_2020/flow/inline.rs

```diff
@@ -2380,7 +2380,7 @@ impl<'a> ContentSizesComputation<'a> {
                     self.containing_block_writing_mode,
                 );
 
-                self.current_line.min_content += self.pending_whitespace + outer.min_content;
+                self.current_line.min_content += outer.min_content;
```

Comment:
I think this might look like a mistake without a comment and the one you made for this PR is great:

```suggestion
                // For the min-content size we should wrap lines wherever is possible,
                // so wrappable spaces shouldn't increase the length of the line,
                // they will just be removed or hang at the end of the line.
                self.current_line.min_content += outer.min_content;
```